packages/desmos-std/src/profiles/mocks.rs

```diff
@@ -0,0 +1,117 @@
+use crate::profiles::{
+    models_app_links::{AppLinkResult, ApplicationLink, CallData, Data, OracleRequest},
+    models_blocks::UserBlock,
+    models_chain_links::{ChainConfig, ChainLink, ChainLinkAddr, Proof, Signature},
+    models_common::PubKey,
+    models_dtag_requests::DtagTransferRequest,
+    models_profile::{Account, Pictures, Profile},
+    models_relationships::Relationship,
+};
+use cosmwasm_std::{Addr, Binary, ContractResult};
+
+pub trait MockQueriesProfiles<T> {
+    fn custom_query_execute(query: &T) -> ContractResult<Binary>;
+}
+
+/**
+This file contains some useful mocks of the Desmos x/profiles modules types ready made to be used
+in any test
+**/
+
+pub fn get_mock_profile() -> Profile {
```

Comment:
Should we wrap all mock functions into a struct?

Reply:
Yes, looks reasonable to do it. It will explicit it more clearly to devs when they need to use them